pom.xml

```diff
@@ -248,6 +248,35 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+					<groupId>org.ec4j.maven</groupId>
+					<artifactId>editorconfig-maven-plugin</artifactId>
+					<version>0.1.3						                        						                        						<!-- check the latest on http://search.maven.org/maven2/org/ec4j/maven/editorconfig-maven-plugin/ --></version>
+					<executions>
+						<execution>
+							<id>check</id>
+							<phase>verify</phase>
+							<goals>
+								<goal>check</goal>
+							</goals>
+						</execution>
+					</executions>
+					<configuration>
+						<!-- See http://ec4j.github.io/editorconfig-maven-plugin/ for full configuration reference -->
+						<excludes>
+							<!-- Note that maven submodule directories and many non-source file patterns are excluded by default -->
+							<!-- see https://github.com/ec4j/editorconfig-linters/blob/master/editorconfig-lint-api/src/main/java/org/ec4j/lint/api/Constants.java#L37 -->
+							<!-- You can exclude further files from processing: -->
+							<exclude>src/main/**/*.whatever</exclude>
+							<exclude>src/test/resources/alma-fix/</exclude>
+						</excludes>
```

Comment:
Why excluding the test files? AFAIR we started this isssue here because of the different formatting of test files, originating from https://github.com/hbz/lobid-resources/pull/1895#issuecomment-2114169914.

Reply:
The testfiles are generated automatically. The pom.xml had the problem that triggered our initiative to use a editor config plugin.

Reply:
"The testfiles are generated automatically." Not necessarily. E.g. you open such an XML in your editor which reformats it automatically . I think that was the case in [66782bb](https://github.com/hbz/lobid-resources/pull/1895/commits/66782bb1c237790a12f159503152ff8ae6bae45e) (which doesnt' exist anymore - but it was there that I complained that diffs are unreadable because every line was changed by reformatting).

Reply:
It formats only when saving. I never save the test xml files.
In the old commit you complained because I changed the heap size in the `pom.xml` and also changed the formatting in the pom.xml at the same time. Because of that I added a separate commit https://github.com/hbz/lobid-resources/pull/1895/commits/214ad1aa20bab1d3a2e1be8f6304b037b02f0da3 before https://github.com/hbz/lobid-resources/pull/1895/commits/6ba89272653eda944fd4bf5476d0f111f2bc8c9c.